src/qml/filters/audio_normalize_1p/ui.qml

```diff
@@ -226,6 +228,25 @@ Item {
             onClicked: maxrateSlider.value = 3
         }
 
+        Label {
+        }
+
+        CheckBox {
+            id: discResetCheckbox
+
+            text: qsTr('Reset on discontinuity')
+            onCheckedChanged: filter.set('discontinuity_reset', checked)
```

Comment:
I believe using `onCheckedChanged` means that when you set `.checked` in `setControls()`, it is going to set the property. Will that be problematic at all? Since this inherits from AbstractButton, you can also use `onClicked` but you will need to change the undo onClicked.

Reply:
> I believe using onCheckedChanged means that when you set .checked in setControls(), it is going to set the property.

Yes
> Will that be problematic at all?

No. Because the property setting is ignored if it is already set to the same value
https://github.com/mltframework/shotcut/blob/master/src/qmltypes/qmlfilter.cpp#L165